thumbservice/tests.py

```diff
@@ -164,7 +164,7 @@ def thumbservice_client():
 def s3_client():
     # This should be passed in to all test functions to mock out calls to aws
     with mock_s3():
-        config = boto3.session.Config(region_name='us-west-2', signature_version='s3v4')
```

Comment:
removed this region name because a later version of moto (what we use to mock s3) broke when specifying it. figured it didn't matter much since these are just tests. I've tested the application with a live dev bucket anyway.